Modules/vc-module-export/VirtoCommerce.ExportModule.Core/Model/ExportedTypeDefinition.cs

```diff
@@ -5,28 +5,28 @@
 namespace VirtoCommerce.ExportModule.Core.Model
 {
     /// <summary>
-    /// Definition of single exported entity type
+    /// Definition of exported entity type
     /// </summary>
     public class ExportedTypeDefinition
     {
         /// <summary>
-        /// Logical type name, given during registration. Commonly non-equal to <see cref="ExportableEntity{T}"/> descendants type names.
-        /// Users can see it in type selector blade.
+        /// Logical type name, given during registration. It could be non-equal to exportable type name.
+        /// Users can see it in a type selector blade.
```

Comment:
Please remove
/// Users can see it in a type selector blade.

It is UI implementation specific